apps/web/src/components/bank-account-form/sections/bank-account-success.tsx

```diff
@@ -0,0 +1,137 @@
+import {
+  GetPaymentBankAccountInstructions,
+  PackType,
+  PublishedPack,
+} from '@algomart/schemas'
+import { CheckCircleIcon } from '@heroicons/react/outline'
+import useTranslation from 'next-translate/useTranslation'
+import { useCallback } from 'react'
+
+import css from './bank-account-success.module.css'
+
+import Button from '@/components/button'
+import Heading from '@/components/heading'
+import { isAfterNow } from '@/utils/date-time'
+import { urls } from '@/utils/urls'
+
+interface BankAccountSuccessProps {
+  bankAccountInstructions: GetPaymentBankAccountInstructions | null
+  release: PublishedPack
+}
+
+export default function BankAccountSuccess({
+  bankAccountInstructions,
+  release,
+}: BankAccountSuccessProps) {
+  const { t } = useTranslation()
+
+  const isActiveAuction =
+    release.type === PackType.Auction &&
+    isAfterNow(new Date(release.auctionUntil as string))
+
+  const handleReturnToListing = useCallback(() => {
+    const path = urls.release.replace(':packSlug', release.slug)
+    if (typeof window !== 'undefined') {
+      window.location.assign(new URL(path, window.location.origin).href)
+    }
```

Comment:
Is using `router.push(path)` not viable here? (where `const router = useRouter()`)

Reply:
Good call! Changed in the other branch with [fd3d2bd](https://github.com/deptagency/algomart/pull/123/commits/fd3d2bd79ade44c23c4d081fd5b9d37e269a3de5)